roles/ovirt-host-deploy-firewalld/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Check if firewalld is installed
+  package:
+    name: firewalld
+    state: present
+
+- name: Check if firewalld is runnning
+  service:
+    name: firewalld
+    state: started
+
+- name: Enable common firewalld rules
+  firewalld:
+    port: "{{ item.port | default(omit) }}"
+    service: "{{ item.service | default(omit) }}"
+    permanent: yes
+    immediate: yes
+    state: enabled
+  with_items:
+    - "{{ host_deploy_firewalld_common }}"
+
+- name: Enable virt firewalld rules
+  firewalld:
+    port: "{{ item.port | default(omit) }}"
+    service: "{{ item.service | default(omit) }}"
+    permanent: yes
+    immediate: yes
+    state: enabled
+  with_items:
+    - "{{ host_deploy_firewalld_virt }}"
+  when: "host_deploy_virt_enabled"
+
+- name: Enable gluster firewalld rules
+  firewalld:
+    port: "{{ item.port | default(omit) }}"
+    service: "{{ item.service | default(omit) }}"
+    permanent: yes
+    immediate: yes
+    state: enabled
+  with_items:
+    - "{{ host_deploy_firewalld_gluster }}"
+  when: "host_deploy_gluster_enabled"
+
+- name: Include cluster specific firewalld rules
+  include_vars: "cluster_version_{{host_deploy_cluster_version}}.yml"
```

Comment:
Shouldn't we also check for virt-cluster-<VERSION>.yml and gluster-cluster-<VERSION>.yml?